CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -1092,14 +1127,26 @@ public boolean NeedsDisplayUpdate() {
     
     @Override
     public int Initialise(CVKDevice cvkDevice) {
+        int ret = VK_SUCCESS;
+        
         this.cvkDevice = cvkDevice;
         
         // This only needs to be initialised once but can't be static as each graph will
         // have their own device and the layout must be bound to that.
         if (hDescriptorLayout == VK_NULL_HANDLE) {
```

Comment:
Those checks against null made sense when the handle was static, they should be replaced with CVKAssert(h == null) now to make sure we aren't double initialising.

Reply:
changed to asserts